examples/ding-dong-bot.ts

```diff
@@ -76,6 +76,15 @@ const bot = WechatyBuilder.build({
   // }
 })
 
+// 使用wechaty-puppet-wechat，需要设置uos: true
```

Comment:
Could you please use English to write the comments because it will be more readable by developers all over the world?

Thank you very much!

Reply:
OK, it's my negligence. It has been revised